react/core/main.tsx

```diff
@@ -109,6 +109,30 @@ function start() {
           state: getState(runtime),
         }))
     } else {
+      const {account: accountName} = window.__RUNTIME__
+
+      const { tagManagerId } = await fetch(
```

Comment:
Better to use axios.
When u use it, `{ headers: { Accept: 'application/json' }` and `.then(res => res.json())` are not necessary

Reply:
use axios